source/content/guides/drupal-9-migration/06-build-tools-to-d9-build-tools.md

```diff
@@ -0,0 +1,281 @@
+
+  Enable deep patching to view the correct MariaDB version in the Dashboard:
+
+   ```bash{promptUser: user}
+   composer config extra.enable-patching true
+   ```
+
+1. Edit `composer.json` and remove `--no-dev` from the `scripts` section to allow the dev dependencies to be available in the integration environment.
+
+  Do not remove the close quote, `"`:
+
+   ```json:title=composer.json
+     "scripts": {
+       "build-assets": [
+         "@prepare-for-pantheon",
+         "composer install --optimize-autoloader --no-dev" //highlight-line
+       ],
+   ```
+
+1. Add `composer-patches` to the `require` list and run `composer update`:
+
+   ```bash{promptUser: user}
+   composer require cweagans/composer-patches drupal/upgrade_status --no-update
+   composer update -W --optimize-autoloader --prefer-dist
+   ```
+
+1. If the site doesn't already have a `pantheon.yml` file, create one with the following values (the comments `#` are optional):
+
+   ```yaml:title=pantheon.yml
+   api_version: 1
+   
+   # Move the DOCUMENT_ROOT of your site to the */web* folder:
+   web_docroot: true
+   
+   # Drupal 9 requires 7.3 or higher. If your code isn't ready for 7.4 you may need to use 7.3 here:
+   php_version: 7.4
+   
+   # Drupal 9 requires a higher version of the DB. It will take a few minutes to complete the upgrade to 10.4 once you push this file:
+   database:
+     version: 10.4
+   
+   # Drupal 9 prefers drush 10. If you have written a lot of custom drush commands you may need to go back to drush 9 or 8:
+   drush_version: 10
+   ```
+
+   An existing `pantheon.yml` file may have more values than this in it, but these are the ones with which we are concerned. If the values already exist in the `pantheon.yml` file, change them to the values in this example.
+
+1. Commit and push the changes:
+
+   ```bash{promptUser: user}
+   git add composer.json composer.lock pantheon.yml config/*
+   git commit -m 'updating to drush 10/mariadb 10.4/config'
+   git push origin d9-upgrade-2021
+   ```
+
+  If all has gone well, you will see something like the following:
+
+   ```bash{outputLines:1-5}
+   remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
+   remote:
+   remote: Create a pull request for 'd9-upgrade-2021' on GitHub by visiting:
+   remote:      {{URL TO YOUR REPOSITORY}}
+   remote:
+   ```
+
+1. Copy the URL from the result (line 4 in the previous output) and use your local web browser to navigate to it to create a pull request. Creating a pull request will cause Build Tools to create an **Integration Environment**. This is called `$ENV` in the next steps.
+
+1. After the build has finished without error, you will see a new environment in the Dashboard under **Multidev** named in reference your pull request.
+
+   ```bash{promptUser: user}
+   terminus drush $SITE.$ENV pm-enable upgrade_status --yes
```

Comment:
@stovak a quick search tells me this line is to enable modules, but that doesn't match with the text before it. Can you give me a "run this to do _this_" for it, please?